lib/tito.rb

```diff
@@ -0,0 +1,29 @@
+require "httparty"
+require_relative "tito/event"
+require_relative "tito/null_event"
+
+class Tito
+  include HTTParty
+
+  base_uri "https://api.tito.io/railsbridge-boston"
+
+  def event_list
```

Comment:
Since we always want only the first event, how about making an `upcoming_event` method that's just `event_list.first`?